extended/src/main/java/io/kubernetes/client/extended/kubectl/Kubectl.java

```diff
@@ -192,6 +196,25 @@ public T namespace(String namespace) {
       this.namespace = namespace;
       return (T) this;
     }
+
+    public T apiGroup(String apiGroup) {
+      this.apiGroup = apiGroup;
+      return (T) this;
+    }
+
+    public T apiVersion(String apiVersion) {
+      this.apiVersion = apiVersion;
+      return (T) this;
+    }
+
+    public T resourceNamePlural(String resourceNamePlural) {
+      this.resourceNamePlural = resourceNamePlural;
+      return (T) this;
+    }
+
+    public boolean isNamespaced() {
```

Comment:
I'm still not sure that this is the right test, since `kubectl` allows "no namespace" to be "current namespace"

Reply:
agree, i moved `isNamespaced` out of resource builder